src/frontend/src/lib/types/token.ts

```diff
@@ -75,6 +75,7 @@ export type TokenUi = Token & TokenFinancialData;
 export type OptionTokenUi = Option<TokenUi>;
 
 export type TokenUiGroup = {
+	id: TokenId;
```

Comment:
it is needed as direct prop of `TokenUiGroup` because when we list the "assets" (either `TokenUi` or `TokenUiGroup`), we index it given the ID, both for the `EACH` loop and for the transitions/animation.

```typescript
let tokens: TokenUiOrGroupUi[] | undefined;
```
```svelte
<TokensDisplayHandler bind:tokens>
	<TokensSkeletons {loading}>
		<div class="mb-3 flex flex-col gap-3">
			{#each tokens ?? [] as token (token.id)}
				<div.....
```

Reply:
But in that case `id: TokenId` is not really correct since a group is a not a token. I guess that's my confusion. Should the `id` be just defined as `symbol` in `TokenUiOrGroupUi`? Does that make sense?

Reply:
I agree with you that the ID should be typed separately. This shortcut was taken because downward elements, like the stores, are typed to tokenid. I suggest adding the type groupId = tokenId and cleaning this up with a to-do.

Reply:
Here we go be666037

Reply:
so, the animation will not understand if they have different id in the list... I imagine @peterpeterparker suggestion was to call the type in a different way, not to give another ID...

is that an acceptable behavior for the design team?

Reply:
My comment was really just about the type not being a token.

Reply:
Ah understood, thank you for the clarification!

@darioloetsc may you please just test that there are no glitches in the animation/transition with the new ID?

Reply:
Done with:

- PR #3115